guide/12-enrich-data-with-thematic-information/part2_enrich_point_study_areas.ipynb

```diff
@@ -0,0 +1,1205 @@
+{
```

Comment:
<p><em>concatenate </em>the components</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>